packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Transactions/Transactions.tsx

```diff
@@ -1,6 +1,7 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import CommunityInfo from '../common/CommunityInfo';
 import { transactionHistoryData } from '../common/sampleData'; // TODO: get data from API
```

Comment:
whats the plan for fetching data from api? do we have ticket for this? do we have pagination? search will be done on front end or backend?

Reply:
i created one https://github.com/hicommonwealth/commonwealth/issues/7059, search will depend on pagination, will discuss that with product